services/update.py

```diff
@@ -81,6 +81,17 @@ def __init__(self, data, compat_mode='strict'):
         self.data['row'] = {}
         self.version_int = 0
         self.compat_mode = compat_mode
+        self.use_json = self.should_use_json()
+
+    def should_use_json(self):
+        # We serve JSON manifests to Firefox and Firefox for Android only,
+        # because we've seen issues with Seamonkey and Thunderbird.
```

Comment:
I thought the Thunderbird issues were resolved and it was just the setting preventing the update check from happening?

Reply:
I don't want to take any chances. I'm not 100% sure where the seamonkey issue is coming from, it might be related to the add-ons we were testing with so just in case, I'm putting thunderbird and seamonkey in the same basket.